### ISSUE TITLE
Error thrown if file time metadata is invalid.

### ISSUE THREAD

**Issue by BrennanMcDonald** [ogham/exa#556]
## Problem
When using a command that reports file time metadata (e.g. `exa -l`) in a directory with files or subdirectories that have invalid (before `SystemTime::UNIX_EPOCH`) we get an error:

```
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)', src/libcore/result.rs:999:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
[1]    84324 abort      exa -l
```

## Expected Behaviour
It is expected that when listing a file with an inaccurate time we would see one of the following:
 - Epoch Time
 - Current Time
 - Null/empty string

## Root Cause
In [/src/fs/file.rs](https://github.com/ogham/exa/blob/89c861f24d18650fc3cef50900e3341b5b038ccf/src/fs/file.rs#L329) there is no check to ensure that `self.metadata.modified()` comes after `UNIX_EPOCH`
```
pub fn modified_time(&self) -> Duration {
    self.metadata.modified().unwrap().duration_since(UNIX_EPOCH).unwrap()
}
```
## Proposed Solution
Adding a check in `file.rs` to ensure that the file's time (modified, accessed, created) falls after `UNIX_EPOCH`

```
pub fn accessed_time(&self) -> Duration {
    if self.metadata.modified().unwrap() < UNIX_EPOCH{
        return SystemTime::now().duration_since(UNIX_EPOCH).unwrap()
    } else {
        return self.metadata.created().unwrap().duration_since(UNIX_EPOCH).unwrap()
    }
}
```

**Comment by ogham** [comment]
The fix for this (#670) has been merged and will go out with the next release.

**Comment by olbrew** [comment]
> The fix for this (#670) has been merged and will go out with the next release.

Do you have an estimation of when this next release would be? I've been seeing this error on a static website directory for over a year. A release to fix this would be really nice! :)

**Comment by ariasuni** [comment]
Dozens of people already asked for a new release, please don’t.

In the meantime, you can use `touch` on the offending files (or build exa from source).
